src/coprocessor/endpoint.rs

```diff
@@ -162,31 +85,14 @@ impl Host {
         engine: Box<Engine>,
         scheduler: Scheduler<Task>,
         cfg: &Config,
-        pd_task_sender: FutureScheduler<PdTask>,
+        pool: ReadPool<ReadPoolContext>,
```

Comment:
Why pass `pool` instead of `pd_scheduler` here?

Reply:
Because `pd_scheduler` is already passed to the `ReadPool` (to build the read pool context). It is no longer required to be passed to `EndpointHost` Please refer to `Storage::new()`'s signature, they are similar.

Reply:
I think @hicqu means we could create pool with `pd_scheduler` in this function.